common/common.go

```diff
@@ -162,8 +162,9 @@ func BlockIngestor(cache *BlockCache, startHeight int, rep int) {
 				"reorg":  reorgCount,
 			}).Warn("REORG")
 			continue
+		} else {
+			reorgCount = 0
 		}
-		reorgCount = 0
 		height++
```

Comment:
I don't love having `else` following `continue` (or `return` or `break`); in this case it's more justified than usual because both paths modify `reorgCount`. But if you don't feel strongly about it, I'd rather not have this change; it's also okay to keep it, just a suggestion.

Reply:
Oh I hadn't noticed the `continue`! That's weird, the behavior I was observing was that `reorgCount` was always getting reset to `0` so the fatal error never fired. I need to investigate more because if I understand `continue` correctly then this change shouldn't change the behavior, but it does.

Reply:
Aha! I was mistaken about the cause of the bug I was seeing. This actually doesn't fix it. What's *actually* happening, when my mock RPC keeps returning the same block for every height, is it oscilates between the cache detecting a reorg and not detecting a reorg, so `height` keeps decrementing and `reorgCount` never exceeds 1.